test/fast-mocha.opts

```diff
@@ -1,5 +1,4 @@
 --exit
 --require ts-node/register/transpile-only
---require coffeescript/register
 --timeout 30000
-test/*.{ts,coffee}
+test/*.{ts,js}
```

Comment:
This configuration previously would not allow you to run *.js test files with the test:fast command. Now that js is added you can run them.